nifi-nar-bundles/nifi-grpc-bundle/nifi-grpc-processors/src/main/java/org/apache/nifi/processors/grpc/ListenGRPC.java

```diff
@@ -85,18 +89,20 @@
             .build();
     public static final PropertyDescriptor PROP_USE_SECURE = new PropertyDescriptor.Builder()
             .name("Use TLS")
-            .displayName("Use TLS")
-            .description("Whether or not to use TLS to send the contents of the gRPC messages.")
+            .displayName("Use SSL/TLS")
+            .description("Whether or not to use SSL/TLS to receive the contents of the gRPC messages.")
```

Comment:
Recommend leaving the existing display name and description since SSLv3 is no longer available for most Java Virtual Machines.  Unfortunately, classes still contain the SSL prefix even though only TLS is allowed.

Reply:
Done.
My original intent was to keep some kind of consistency with the name of the 'SSL Context Service' property.
But it is better to go ahead and use TLS where possible.